examples/event_handling/resample.py

```diff
@@ -1,9 +1,15 @@
 """
-========
-Resample
-========
+===============
+Resampling Data
+===============
+
+
+Downsampling lowers the sample rate or sample size of a signal. In
+this tutorial, the signal is downsampled when the plot is adjusted
+through dragging and zooming.
```

Comment:
re worked this a bit.

It works on zoom, but not pan which seems like a bug Mpl, not in the example.